Specs/Scene/TilesetMetadataSpec.js

```diff
@@ -417,7 +427,7 @@ describe("Scene/TilesetMetadata", function () {
           position: {
             type: "VEC3",
             componentType: "FLOAT32",
-            optional: true,
+            required: false,
```

Comment:
I see some others occurrences of `optional: true` in `MetadataTableSpec.js` and `TilesetMetadataSpec.js`. Can those be updated now or are they waiting for the future required/no-data PR?

Reply:
ah yeah I had just missed some spots